readme.md

```diff
@@ -127,18 +136,28 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** What are the arguments for `Model.updateOne()`?
 
 * **Your Answer:**
+Filter - The selection criteria.
+Update - Your modifications
+Options - What else you may want to include
+
 
 * **Question:** The response you receive is _not_ the document you updated. What information is being represented here? Try replacing `Model.updateOne()` with the [Model.findOneAndUpdate()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndUpdate) and see the difference in the result.
 
 * **Your Answer:**
+It added the placeholder text that we has added in title.
```

Comment:
Not sure what you mean by this. It should have returned information about the query itself.